conda-forge.yml

```diff
@@ -1,3 +1,5 @@
+bot:
+  inspection: update-grayskull
```

Comment:
Same as https://github.com/conda-forge/gotrue-feedstock/pull/9/files#r1349536520. This is just a recommendation.  I can remove the last commit if you want.  But this will automatically update the dependencies for each new release by using the [grayskull](https://github.com/conda/grayskull) tool.